lua-plugins/cli_filter.lua.in

```diff
@@ -17,6 +17,29 @@
 local git_date = '$Date$'
 local git_revision = '$Revision$'
 
+-- Access constraints mapping to partition
+local constraints_mapping = {
```

Comment:
Do we add podman and also the quantum partition or leave it for a later PR?

Reply:
This is something I did think about, but they do not exist in the ReFrame config so I didn't include them.

Reply:
Then I would say, let's leave it for a later PR if @halfflat is fine with that.